lib/error_tracker.ex

```diff
@@ -107,6 +107,11 @@ defmodule ErrorTracker do
   * A `{kind, exception}` tuple in which case the information is converted to
   an Elixir exception (if possible) and stored.
   """
+
+  @type exception :: struct() | {atom(), struct()}
+  @type stack_trace :: [{atom(), fun(), integer(), Keyword.t()}]
+
+  @spec report(exception(), stack_trace()) :: %Occurrence{} | :noop
```

Comment:
I'd rather use `Occurrence.t` which is the convention used by popular libraries and Elixir standard library.